_rules/first-focusable-element-is-link-to-main-8a213c.md

```diff
@@ -0,0 +1,728 @@
+---
+id: 8a213c
+name: First focusable element is link to main content
```

Comment:
Keeping this rule as Best Practice given that it was already written and that both Siteimprove and Qualweb have expressed interest in implementing it anyway.